SCIBarcodeScanner/Resources/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,5 @@
+"camera.authorization.title" = "Camera access";
+"camera.authorization.reason" = "In order for the barcode scanner to work, please allow access to the camera in the settings.";
+"camera.authorization.settings" = "Settings";
+"generic.cancel" = "Cancel";
```

Comment:
Should we really use these MIOesque keys? (See main comment in delegate.)

Reply:
I changed them to be general